### PR TITLE
PUT /api/zoops/:id

### LINKED ISSUE #11: PUT /zoops/:id
Edit an existing Zoop (if I wrote it)

### PR DESCRIPTION
**Pull request by dyazdani**
## Description
PUT /api/zoops/:id endpoint. Made a TODO to add auth middleware once the PRs that set that up are merged.

## Related Issue

Closes #11 

## Checklist
- [x] Created tests which fail without the change (if possible)
- [x] Confirmed all tests are passing

## How Has This Been Tested?

Tested with Postman.
Screen Shot 2023-10-17 at 15 40 02

### CODE REVIEW

File: .vscode/settings.json
Comment (paloobi) [inline .vscode/settings.json:1]: Delete me. Add to gitignore

Comment (paloobi) [reply .vscode/settings.json:1]: Generally, it's a good idea to add all of `.vscode` to gitignore

File: src/api/zoops.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                authorId,
+                receiverId
+            }
+        })
+        res.send({zoop});
+})
+
+// PUT /api/zoops/:id
+//TODO: add auth middleware function when auth.ts file is available for import
+zoopsRouter.put("/:id", async (req, res, next)=> {
```
Comment (paloobi) [inline src/api/zoops.ts:35]: nit: It feels like this is really a PATCH since it is just updating the content? but it's up to you

**Review by paloobi — APPROVED** [review]

File: src/api/zoops.ts (hunk 1 above)
Comment (dyazdani) [reply src/api/zoops.ts:35]: I'll make an issue to change it if there is time. I only did PUT because that was what was in the Zoop requirements doc.

**Pull request merged**